resolvers/event.js

```diff
@@ -41,17 +41,18 @@ const eventResolvers = {
       // Check if createdBy is valid
       const user = await db.user.findByPk(args.createdBy);
       if (!user) {
-        throw new Error("Invalid user ID");
+        throw new Error('Invalid user ID');
       }
 
-      await db.event.create({
+      const newEvent = await db.event.create({
         name: args.name,
         eventDate: args.eventDate,
         createdBy: args.createdBy,
         isActive: args.isActive,
       });
```

Comment:
i prefer using promise chaining over awaits - can you change this to something along these lines
```suggestion
      return db.event.create({
        name: args.name,
        eventDate: args.eventDate,
        createdBy: args.createdBy,
        isActive: args.isActive,
      }).then((newEvent) =>{
        name: newEvent.name,
        eventDate: newEvent.eventDate,
        createdBy: newEvent.createdBy,
        isActive: newEvent.isActive,
        ambulances: [],
        hospitals: [],
      });
```